lib/pure/collections/sets.nim

```diff
@@ -730,17 +730,19 @@ proc incl*[A](s: var OrderedSet[A], key: A) =
 
   inclImpl()
 
-proc incl*[A](s: var HashSet[A], other: OrderedSet[A]) =
-  ## Includes all elements from the OrderedSet `other` into
-  ## HashSet `s` (must be declared as `var`).
+proc incl*[T](s: var SomeSet1[T], other: SomeSet2[T]) =
+  ## Includes all elements from the right hand side set `other` into
+  ## left hand side set `s` (must be declared as `var`).
   ##
   ## See also:
+  ## * `excl proc <#excl,HashSet[A],A>`_ for excluding an element
   ## * `incl proc <#incl,OrderedSet[A],A>`_ for including an element
+  ## * `containsOrIncl proc <#containsOrIncl,HashSet[A],A>`_
   ## * `containsOrIncl proc <#containsOrIncl,OrderedSet[A],A>`_
   runnableExamples:
     var
       values = toHashSet([1, 2, 3])
-      others = toOrderedSet([3, 4, 5])
+      others = toHashSet([3, 4, 5])
```

Comment:
If SomeSet1/2 can be either of the 2 set types, why does this test need to change?

Reply:
This is some artifact caused by trying different implementations. Before I wrote the generic on the set type procedure first I stated to write four different procedures. I will revert it.

Reply:
It's fixed.